packages/msgraph/src/Adaptor.js

```diff
@@ -213,21 +213,21 @@ export function getFolder(pathOrId, options, callback = s => s) {
  * @return {Operation}
  */
 export function getFile(pathOrId, options, callback = s => s) {
+  const defaultOptions = {
```

Comment:
Oh, I hadn't intended that `getFile` would expose `parseAs` to the job. Interesting, let me think about that!

Files usually have an extension and a mimetype. So when we request a file we should always be able to work out the type and return an appropriate data structure to represent it.

So I really don't think we should expose this option. Instead, we should hard-code `parseAs` into the request (a bit like how we pass auth differently).

Also, we're introducing `parseAs` to `request` as a temporary fix. But if we expose it in `getFile`, we're kinda committed to maintaining it in future. Which I'm not comfortable with.

Reply:
I like the idea of file mime type but i think it's going to be at-least a minor version bump and it needs planning

Reply:
Oh I'm not suggesting we do any of the mime stuff now. I'm just saying we should return files as plain text for now!